### ISSUE TITLE
No support for compressed plots

### ISSUE THREAD

**Issue by softwareuser99** [STATION-I/stai-blockchain#30]
**Describe the bug**
With the introduction of compressed plots and many people replotting, STAI needs to support compressed plots in order to still leverage existing CHIA farms. Are there plans to update the STAI software to include that support?

Sorry, not a bug per se but all other links are brokend..

**Comment by WarutaShinken** [comment]
> With the introduction of compressed plots and many people replotting, STAI needs to support compressed plots in order to still leverage existing CHIA farms. Are there plans to update the STAI software to include that support?

Yes, we will be experimenting with forking Chia's beta builds for compressed plot support in the near future.

> Sorry, not a bug per se but all other links are brokend..

Which links?

**Comment by AlexandrBursak** [comment]
I think these links on the page "New Issue":
![image](https://github.com/STATION-I/stai-blockchain/assets/4108839/1b92075b-05ce-4d7b-a886-9a6f511a4780)

**Comment by WarutaShinken** [comment]
Compressed plot support will be coming next update. As for the links, I will fix those. Cheers.

**Comment by WarutaShinken** [comment]
New issue page links have been revamped.

**Comment by AlexandrBursak** [comment]
@WarutaShinken 
Any updates related to compressed plots?

**Comment by WarutaShinken** [comment]
Yes, an unofficial deadline of January 31 2024 was set.
